aspnetcore/security/cors/3.1sample/Cors/WebAPI/wwwroot/js/MyJS.js

```diff
@@ -0,0 +1,21 @@
+﻿function MyTestCors3(host, uri, methodName, includeHeaders=false) {
+    const resultSpan = document.getElementById('result');
+    const myHeaders = includeHeaders ? { 'Content-Type': 'x-custom-header' } : {};
+
+    fetch(host + uri,
+        {
+            method: methodName,
+            headers: myHeaders,
+
+        }).then(response => {
+            if (response.ok) {
+                response.text().then(text => {
+                    resultSpan.innerText = text;
+                });
+            }
+            else {
+                resultSpan.innerText = response.status;
+            }
+        })
+        .catch(error => resultSpan.innerText = 'See F12 Console for error');
```

Comment:
`error` is never used, so it's best to do the following:

```suggestion
        .catch(() => resultSpan.innerText = 'See F12 Console for error');
```